### DISCUSSION TITLE
Possible to controll Rethink's inbuilt Wireguard client with Tasker?

### DISCUSSION THREAD

**Discussion by Kamika242** [celzero/rethink-app#1149] [Q&A]
Hi there,

awesome app! Ist there a way to control the inbuilt Wireguard client with Tasker? I don't need the Wireguard connection in my home network, but when I leave home. With the standard Wireguard client it's perfectly possible, example:

Leaving my home network  - > Tasker

start application "Wireguard"
wait 1sec
WireGuardSetTunnel(true,TunnelName)
show homescreen 0

Returning to my home network - > Tasker

start application "Wireguard"
wait 1sec
WireGuardSetTunnel(false,TunnelName)
show homescreen 0

Please, I need the specific commands to open the Wireguard section in the Rethink app and the appropriate commands to start/stop the Wireguard tunnel.

Best regards,

Kamika242

**Comment by ignoramous** [comment]
> Ist there a way to control the inbuilt Wireguard client with Tasker?

Unfortunately, no. Tasker integration is not a priority, but code changes welcome: #63
